.circleci/config.yml

```diff
@@ -50,6 +50,11 @@ jobs:
           command: |
             pipenv run "flake8 art/art"
 
+      - run:
+         name: django-nose
+         command: |
+           pipenv run "nosetests"
+
```

Comment:
let's name this `run tests` and use the default command `python manage.py test` instead of `nosetests` to avoid any flakiness.

Reply:
@nelmandela this too :)

Reply:
I made those changes but I don't know why it's not updated.let me figure out what's happening